services/worker/src/worker/config.py

```diff
@@ -186,6 +187,7 @@ class ParquetAndInfoConfig:
     blocked_datasets: List[str] = field(default_factory=get_empty_str_list)
     commit_message: str = PARQUET_AND_INFO_COMMIT_MESSAGE
     committer_hf_token: Optional[str] = PARQUET_AND_INFO_COMMITTER_HF_TOKEN
+    fully_converted_datasets: List[str] = field(default_factory=PARQUET_AND_INFO_FULLY_CONVERTED_DATASETS.copy)
```

Comment:
it's clever and better than what I did in other parts (`get_empty_str_list`) because the fact that the list is empty or not is specified in the constant. I think we might want to refactor this

Reply:
created https://github.com/huggingface/datasets-server/issues/1522